lib/Common/PlatformAgnostic/DateTimeInternal.h

```diff
@@ -78,13 +78,14 @@ namespace DateTime
         bool fHiResAvailable;
 
         HiresTimerPlatformData() :
-            fInit(false),
             dBaseTime(0),
-            baseMsCount(0),
-            fHiResAvailable(true),
             dLastTime(0),
             dAdjustFactor(1),
-            fReset(true)
+            baseMsCount(0),
+            freq(0),
+            fReset(true),
+            fInit(false),
+            fHiResAvailable(true)
```

Comment:
was there a warning about initialization order or is this just drive-by cleanup?

Reply:
There's one in razzle prefast, and it helps for checking that there's no other uninitialized values when they're in the right order. We do not have initialization order warnings enabled in our jenkins runs.